dev-libs/libical/libical-2.1.0~.recipe

```diff
@@ -0,0 +1,83 @@
+SUMMARY="An implementation of iCalendar protocols and data formats"
+DESCRIPTION="Libical is an Open Source implementation of the iCalendar \
+protocols and protocol data units. The iCalendar specification describes how \
+calendar clients can communicate with calendar servers so users can store \
+their calendar data and arrange meetings with other users."
+HOMEPAGE="http://libical.github.io/libical/"
+COPYRIGHT="Eric Busboom"
+LICENSE="MPL v1.0
+	GNU LGPL v2.1"
+REVISION="1"
+scrGitRev="1509e29de73044856271b18f1024acdee3d6f8ac"
+SOURCE_URI="https://github.com/libical/libical/archive/$scrGitRev.tar.gz"
+CHECKSUM_SHA256="3d4b02ff991916fee19badf6c0117627b6643fbdc1558e216e812108ab40fd9e"
+SOURCE_DIR="libical-$scrGitRev"
+
+ARCHITECTURES="!x86_gcc2 ?x86 ?x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 ?x86"
+
+PROVIDES="
+	libical$secondaryArchSuffix = $portVersion
```

Comment:
Are this and the libical 2 compatible at the ABI level?
If they are not, you need to have different package names (libical and libical2) and proper sonames/compat declarations (see my comments on the other recipe). Otherwise, the two packages will conflict and it will not be possible to install them side by side.
You can look at the libpng recipes for an example of this.